index.html

```diff
@@ -0,0 +1,241 @@
+          <li>
+            <img src="images\projects\img-5.jpeg" alt="Project-five"> 
+          </li>
+          <li>
+            <img src="images\projects\img-6.jpg" alt="Project-six">
+          </li>
+        </ul>
+      </section>
+      <section class="block">
+        <div class="title-area block-center">
+          <h5>Testimonial</h5>
+          <h3>Clients Say About Us</h3>
+        </div>
+        <div class="flex-box">
+          <img src="images\testimonial\testimonial-1.jpg" alt="Testimonial">
+          <div class="slider-info">
+            <img src="images\quote.png" alt="Quote">
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit.
+            Voloptatibus quia, nulla! Maiores et perferendis eaque,
+            exercitationem praesentium nihil.
+            </p>
+            <h4>Jonathan Reinink</h4>
+            <span>CEO</span>
+          </div>
+        </div>
+      </section>
+      <section class="block">
+        <div class="letter-box block-center">
+          <div class="title-area">
+            <h5>Newsletter</h5>
+            <h3>Get Gardening Tips</h3>
+          </div>
+          <div class="mail-box">
+            <input type="email" placeholder="Enter Your Email Address">
+            <input type="button" value="Subscribe" class="btn btn-primary">
+          </div>
+        </div>
+      </section>
+    </div>
+  </main>
+  <hr>
+  <footer id="footer">
+    <div class="container">
+      <ul class="flex-box">
+        <li>
+          <a href class="footer-logo">
+            <img src="images\logo.png" alt="logo">
+          </a>
+          <p>There is no one who loves pain itself, who seeks after it and wants to have it, simply because it is pain</p>
+        </li>
+        <li>
+          <h4>Useful Links</h4>
+          <ul class="useful-links">
+            <li><a href="">About Us</a></li>
+            <li><a href="">Services</a></li>
+            <li><a href="">Contact</a></li>
+          </ul>
+        </li>
+        <li>
+          <h4>Contact</h4>
+          <ul>
+            <li>Live Chat</li>
+            <li>+89009977-776</li>
+            <li>gardeen@contact.uc</li>
+            <li>Facebook Page</li>
+            <li>Instagram</li>
+          </ul>
+        </li>
+        <li>
+          <h4>Contact</h4>
+          <ul class="contact-list">
+            <li><span class="day flex-box">Saturday <span>8.30 am-4.00 pm</span></span></li>
+            <li><span class="day flex-box">Sunday <span>8.30 am-4.00 pm</span></span> </li>
+            <li><span class="day flex-box">Monday<span>8.30 am-4.00 pm</span></span></li>
+            <li><span class="day flex-box">Tuesday<span>8.30 am-4.00 pm</span></span></li>
+            <li><span class="day flex-box">Wednesday <span>8.30 am-4.00 pm</span></span></li>
+            <li><span class="day flex-box">Friday<h5>Closed</h5></span></li>
+          </ul>
+        </li>
+      </ul>
+    </div>
+  </footer>
+  <hr>
+  <div class="copyright">
```

Comment:
Still copyright div outside the footer tag. Please refer to the previous comments.